src/Interface.ts

```diff
@@ -0,0 +1,26 @@
+export type Recipe = {
+    bookmarked: boolean;
+    bought: boolean;
+    recipe: RecipeDetails;
```

Comment:
Maybe would rename to `recipeDetails` since the parent is already called recipe for clarity (was confused when reading `recipe.recipe` above)

Reply:
The properties of Recipe are defined by the API, so I don't think they can be renamed (it errored when I tried). Should the interface name be different?